### PR TITLE
DICEZ token information

### PR DESCRIPTION
**Pull request by bennyarcher**
Transaction

https://scan.thundercore.com/transactions/0x0fd1abe5996b8a92ea275a51068d1c98bd3b85073457878dd9182fc4eeb15563

### CODE REVIEW

File: src/thundercore.json [hunk 1]
```diff
@@ -113,6 +113,17 @@
       ],
       "tags": []
     },
+    {
+      "address": "0xFCDEa9f405bc6bd85C2e90c1DEbB9F3ca7a59D8f",
+      "symbol": "DICEZ",
+      "name": "DICEZ",
```
Comment (anthony-tt) [inline src/thundercore.json:119]: Token name isn't aligned with contract?

**Review by anthony-tt — CHANGES_REQUESTED** [review]

File: src/thundercore.json (hunk 1 above)
Comment (bennyarcher) [reply src/thundercore.json]: Updated the name

Comment (bennyarcher) [reply src/thundercore.json]: By the way, the file thundercore.json has some other issues. The last token section of RAM contains information of Oolong as well with duplicate keys.

Comment (anthony-tt) [reply src/thundercore.json]: Thanks for the reminder, we've noticed that and fixed it.

**Review by anthony-tt — APPROVED** [review]

**Pull request merged**